banyand/stream/part_metadata.go

```diff
@@ -32,8 +32,7 @@ type partMetadata struct {
 	BlocksCount           uint64 `json:"blocksCount"`
 	MinTimestamp          int64  `json:"minTimestamp"`
 	MaxTimestamp          int64  `json:"maxTimestamp"`
-	// TODO: remove this?
-	ID uint64 `json:"-"`
+	ID                    uint64 `json:"-"`
```

Comment:
You should raise another PR to remove it and the relevant "partID" field since the stream doesn't have to duplicate data.